tuxemon/states/techniques/__init__.py

```diff
@@ -25,7 +24,9 @@ class TechniqueMenuState(Menu[Technique]):
     background_filename = "gfx/ui/item/item_menu_bg.png"
     draw_borders = False
 
-    def __init__(self) -> None:
+    def __init__(self, monster: Optional[Monster] = None) -> None:
```

Comment:
Sorry, do we expect TechniqueMenuState to always have a monster? In that case it should be 
`def __init__(self, monster: Monster) -> None:`

Reply:
Yeah, always. No problem, I'm going to address it